src/ext/oblivc/obliv_bits.c

```diff
@@ -1842,6 +681,7 @@ void feedOblivInputs(OblivInputs* spec, size_t count, int party)
     { __obliv_c__##ot rv; \
       OblivInputs spec; \
       setupObliv##tname(&spec,&rv,v); \
+      spec.src_f = v; \
```

Comment:
Although it may seem that way, that is not the behavior I observe. It's been a while since I wrote that segment but I believe that the assignment to ```src_f``` is harmless exactly because it is unioned with ```src```; I am just "placing data" in a bucket that can either be interpreted as a float or as an int; nothing is being overwritten (and the ```setupObliv##tname``` functions, which all call ```__obliv_c__setupOblivBits```, were going to do that anyways). In any case, in my testing I have not encountered any issues with this.